.github/workflows/build.yml

```diff
@@ -0,0 +1,13 @@
+on: pull_request
+name: Build
+jobs:
+  build:
+    name: Build
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Xcode select
+        run: sudo xcode-select -s /Applications/Xcode_11.2.app/Contents/Developer
```

Comment:
so this step is telling the CI process which version of xcode it should use to build? we can assume that the CI server has this copy of xcode at this path?

Reply:
Correct, you can see the list of installed software here: https://github.com/actions/virtual-environments/blob/master/images/macos/macos-10.15-Readme.md